disconnected/about-installing-oc-mirror-v2.adoc

```diff
@@ -52,6 +52,9 @@ include::modules/oc-mirror-building-image-set-config-v2.adoc[leveloffset=+2]
 
 * xref:../updating/understanding_updates/intro-to-updates.adoc#update-service-about_understanding-openshift-updates[About the OpenShift Update Service]
 
+// oc-mirror workflow comparison table
```

Comment:
Just an FYI about Commented-out text is not captured in DITA in [this article](https://spaces.redhat.com/spaces/OSDOCS/pages/654185635/Content+updates+in+preparation+for+migrating+to+DITA#ContentupdatesinpreparationformigratingtoDITA-Additionalmigrationconsiderations)